packages/verified-fetch/src/utils/parse-url-string.ts

```diff
@@ -89,30 +97,43 @@ export async function parseUrlString ({ urlString, ipns, logger }: ParseUrlStrin
   let cid: CID | undefined
   let resolvedPath: string | undefined
   const errors: Error[] = []
+  let resolveResult: IPNSResolveResult | DNSLinkResolveResult | undefined
+  let ttl: number = 5 * 60 // 5 minutes based on https://github.com/ipfs/boxo/issues/329#issuecomment-1995236409
```

Comment:
I'd leave setting the default value to the `setCacheControlHeader` to avoid having it in two places.